manifests/t/Texlive/Texlive/2023/Texlive.Texlive.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Texlive.Texlive
+PackageVersion: 2023
+Installers:
+- InstallerUrl: https://ftp.rrzn.uni-hannover.de/pub/mirror/tex-archive/systems/texlive/tlnet/install-tl-windows.exe

```

Comment:
```suggestion
- InstallerUrl: https://mirrors.rit.edu/CTAN/systems/texlive/tlnet/install-tl-windows.exe
```

Reply:
This ISV doesn't appear to have their own hosting. Their download page links to this `CTAN` mirror, which I believe makes it the ISV's release location, meeting [Policy 1.1.4](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#11-distinct-function--value-accurate-representation).

Reply:
> Hmm, are you getting a different a different download URL when downloading the Texlive installer? For me, the official download link on https://tug.org/texlive/windows.html links to https://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe . But when downloading the installer through Firefox and copying the link to the downloaded file, it seems the file was actually downloaded from ftp.rrzn.uni-hannover.de/pub/mirror/tex-archive, which I chose as the InstallerUrl in wingetcreate.

Yes, my download is coming directly from https://mirrors.rit.edu/CTAN/systems/texlive/tlnet/install-tl-windows.exe - I'm in WA State, so I might be geo-routed to the closer mirror in NY, while you might be geo-routed to the mirror closer to you in Germany. 

> Either way, my original manifest failed the checksum validation for reasons unbeknownst to me so I accepted your changes.

Checksum changes are frequently due to bug or vulnerability fixes, including upstream fixes. Updating the checksum in the manifest is usually the best fix for a mismatch.